testing/architecture-test/src/changes/accepted-changes/accepted-public-api-changes.json

```diff
@@ -88,6 +108,18 @@
                 "Method has been removed"
             ]
         },
+        {
+            "type": "org.gradle.api.ResolutionProvider",
+            "member": "Class org.gradle.api.ResolutionProvider",
+            "acceptation": "Making ResolutionProvider public API",
+            "changes": []
+        },
+        {
+            "type": "org.gradle.api.ResolutionProvider",
+            "member": "Method org.gradle.api.ResolutionProvider.getResolutions()",
+            "acceptation": "Making ResolutionProvider public API",
+            "changes": []
+        },
```

Comment:
I am also surprised to be seeing these. Are we missing the `@since` on both the class and the method?